assets/controllers/select/default_controller.js

```diff
@@ -79,7 +79,7 @@ function transMlang(text) {
         return "";
     }
 
-    var openingTag = "{[ \t]*mlang[ \t]*" + Translator.locale + "[ \t]*(default)?[ \t]*}";
+    var openingTag = "{[ \t]*mlang[ \t]+" + Translator.locale + "([ \t]+default)?[ \t]*}";
```

Comment:
Why move the [ \t]+ inside the (default) parentheses?

Reply:
We want to accept:
```
{mlang en}
{mlang en }
```
If `[ \t]+ was not inside `()`, `{mlang en} wouldn't be supported, because regex expects at least one space/tab.

Please consider